cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -18,20 +18,23 @@ package controlplane
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"reflect"
 	"sort"
 	"strings"
 	"testing"
 
+	"github.com/lithammer/dedent"
+
 	"k8s.io/apimachinery/pkg/util/sets"
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
-	authzmodes "k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes"
-
+	staticpodutil "k8s.io/kubernetes/cmd/kubeadm/app/util/staticpod"
 	testutil "k8s.io/kubernetes/cmd/kubeadm/test"
+	authzmodes "k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes"
```

Comment:
this reference was just removed in https://github.com/kubernetes/kubernetes/pull/80307, avoid reintroducing it

Reply:
Good catch. unfortunately https://github.com/kubernetes/kubernetes/pull/80307 removed this reference from `manifests.go` but not from `manifests_test.go`
I opened https://github.com/kubernetes/kubeadm/issues/1713 to track this cleanup